float64 finger0_param
float64 finger1_param
---
int8 return_code
